04_numeronym/hsy3418/main_test.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+)
+
+func TestNumeronym(t *testing.T) {
+
+	actual := numeronyms("accessibility", "Kubernetes", "abc", "Android Enterprise", "", "ab")
+	expected := []string{"a11y", "K8s", "a1c", "A16e", "", "ab"}
```

Comment:
As @pfannerj has said, "abc" should stay as "abc", not "a1c". Please fix the test case and the code.